ledger/streaming/api/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "mc-ledger-streaming-api"
+version = "1.3.0-pre0"
+authors = ["MobileCoin"]
+license = "MIT OR Apache-2.0"
+links = "mc-ledger-streaming-api"
+build = "build.rs"
+edition = "2018"
+
+[dependencies]
+mc-api = { path = "../../../api" }
+mc-common = { path = "../../../common" }
+mc-consensus-scp = { path = "../../../consensus/scp" }
+mc-transaction-core = { path = "../../../transaction/core" }
+
+displaydoc = { version = "0.2", default-features = false }
+futures = { version = "0.3.21" }
+grpcio = { version = "0.9.0" }
+protobuf = "2.22.1"
+
+[build-dependencies]
+mc-util-build-grpc = { path = "../../../util/build/grpc" }
+mc-util-build-script = { path = "../../../util/build/script" }
+
+cargo-emit = "0.2.1"
```

Comment:
Should we merge the cargo.toml so that there's only 1 for streaming?

Reply:
I've opted to keep them separate since we expect the server(s) and clients to live in different binaries, with different deps.